apps/site/pages/docs/performance.mdx

```diff
@@ -0,0 +1,48 @@
+<header>
```

Comment:
I don't think this document here is part of the faststore framework, but of the faststore cloud and the cloud documentation is not on this site.

Reply:
Should I remove it, then?

Reply:
I believe so